planetiler-core/src/main/java/com/onthegomap/planetiler/config/Bounds.java

```diff
@@ -52,11 +55,17 @@ public Bounds addFallbackProvider(Provider latLonProvider) {
     return this;
   }
 
+  public Bounds setShape(MultiPolygon shape) {
+    this.shape = shape;
+    set(shape.getEnvelopeInternal());
```

Comment:
Would it ever make sense to set a shape and bounding box? If so would we take the union? intersection?